schema/omega-services.async.api.yaml

```diff
@@ -1,525 +1,480 @@
+      name: InvalidApplicationError
+      title: Invalid Application Error
+      description: >-
+        The client application as identified by the `OMGApplicationID` header is
+        either invalid, not-recognised, or prohibited.
+      traits:
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/JMSResponse'
+        - $ref: '#/components/messageTraits/OMGMessage'
       payload:
-        $ref: "#/components/schemas/editSetRecord"
-      correlationId:
-        description: Default Correlation ID
-        location: $message.header#/correlationId
-
-    updateResponseStatus:
-      name: updateResultStatus
-      title: Update Result Status
+        type: array
+        items:
+          $ref: '#/components/schemas/ErrorMessage'
+
+    AuthenticationError:
+      messageId: OELIFERR004
+      name: AuthenticationError
+      title: Authentication Error
+      description: >-
+        Authentication failed, likely due to an invalid token being present in
+        the `OMGToken` header.
+      traits:
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/JMSResponse'
+        - $ref: '#/components/messageTraits/OMGMessage'
       payload:
-        $ref: "#/components/schemas/updateResponseStatus"
-      correlationId:
-        description: Default Correlation ID
-        location: $message.header#/correlationId
-
-    errorResponse:
-      name: errorResponse
-      title: Error response message
+        type: array
+        items:
+          $ref: '#/components/schemas/ErrorMessage'
+
+    UnrecognisedMessageTypeError:
+      messageId: OELIFERR005
+      name: UnrecognisedMessageTypeError
+      title: Unrecognised Message Type Error
+      description: >-
+        The value of `OMGMessageTypeID` header was not recognised and so the
+        message cannot be further dispatched for processing.
+      traits:
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/JMSResponse'
+        - $ref: '#/components/messageTraits/OMGMessage'
+      payload:
+        type: array
+        items:
+          $ref: '#/components/schemas/ErrorMessage'
+
+    InvalidMessageError:
+      messageId: OELIFERR006
+      name: InvalidMessageError
+      title: Invalid Message Error
+      description: >-
+        The content of the body of the message has failed validation, e.g. a
+        parsed JSON message may be missing some required properties, or those
+        properties may have invalid values.
+      traits:
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/JMSResponse'
+        - $ref: '#/components/messageTraits/OMGMessage'
+      payload:
+        type: array
+        items:
+          $ref: '#/components/schemas/ErrorMessage'
+
+    ProcessingError:
+      messageId: OELIFERR007
+      name: ProcessingError
+      title: ProcessingError
```

Comment:
Missing a space. Should be: `title: Processing Error`

Reply:
I have appended a commit to address this.